data-raw/rs_supprs_onco_imwg.R

```diff
@@ -0,0 +1,123 @@
+  "01-702-1082",    16L,      "A3", "OVRLRESP", "Overall Response",     "CR",      "CR",      NA,        NA, "INVESTIGATOR",         9,         "WEEK 12", "2013-11-17",   84L,
+  "01-703-1076",     7L,      "A2", "OVRLRESP", "Overall Response",     "PR",      "PR",      NA,        NA, "INVESTIGATOR",         7,          "WEEK 6", "2013-12-04",   42L,
+  "01-703-1076",    16L,      "A3", "OVRLRESP", "Overall Response",     "NE",      "NE",      NA,        NA, "INVESTIGATOR",         9,         "WEEK 12", "2013-12-24",   84L
+  )%>%
+  mutate(
+    STUDYID = "CDISCPILOT01",
+    DOMAIN = "RS",
+    RSCAT = "IMWG",
+    RSEVAL = "INVESTIGATOR"
+  ) %>%
+  relocate(
+    STUDYID, DOMAIN, USUBJID, RSSEQ, RSLNKGRP, RSTESTCD, RSTEST, RSCAT, RSORRES,
+    RSSTRESC, RSSTAT, RSREASND, RSEVAL, VISITNUM, VISIT, RSDTC,
+    RSDY
+  )
+
+
+usethis::use_data(rs_onco_imwg, overwrite = TRUE)
+
+
+supprs_onco_imwg <- tibble::tribble(
+       ~USUBJID, ~IDVARVAL,     ~QNAM,                            ~QLABEL,        ~QVAL,
+  "01-701-1015",        7L,   "PDOFL",       "Progressive Disease: Other",          "Y",
+  "01-701-1015",        7L, "DTHPDFL", "Death Due to Progressive Disease",          "Y",
+  "01-701-1097",        7L,   "PDOFL",       "Progressive Disease: Other",          "Y",
+  "01-701-1097",        7L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-02-10",
+  "01-701-1115",        7L,   "PDIFL",     "Progressive Disease: Imaging",          "Y",
+  "01-701-1146",        7L,   "PDOFL",       "Progressive Disease: Other",          "Y",
+  "01-701-1148",        7L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-02-07",
+  "01-701-1148",       16L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-02-07",
+  "01-701-1148",       26L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-02-07",
+  "01-701-1148",       36L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-02-07",
+  "01-701-1302",        7L,   "PDIFL",     "Progressive Disease: Imaging",          "Y",
+  "01-701-1345",       26L,   "PDOFL",       "Progressive Disease: Other",          "Y",
+  "01-701-1287",       26L,   "PDOFL",       "Progressive Disease: Other",          "Y",
+  "01-701-1287",        7L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-05-30",
+  "01-701-1287",       16L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-05-30",
+  "01-701-1287",       26L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-05-30",
+  "01-701-1287",       36L,  "NACTDT",     "New Anti-Cancer Therapy Date", "2014-05-30",
+  "01-701-1287",       36L,   "PDOFL",       "Progressive Disease: Other",          "Y",
+  "01-702-1082",        7L,   "PDOFL",       "Progressive Disease: Other",          "Y"
+  )%>%
+  mutate(
+    STUDYID = "CDISCPILOT01",
+    RDOMAIN = "RS",
+    IDVAR = "RSSEQ",
+    QORIG = "CRF",
+    IDVARVAL=as.character(IDVARVAL)
+  ) %>%
+  relocate(
+    STUDYID, RDOMAIN, USUBJID, IDVAR, IDVARVAL, QNAM, QLABEL, QVAL, QORIG
+  )
+
```

Comment:
Please could you label the dataset?  See [here](https://github.com/pharmaverse/pharmaversesdtm/blob/cff736346da2b636ccd76814441d7c977aa2ba97/data-raw/dm.R#L15) as an example